forte/data/extractor/feature.py

```diff
@@ -0,0 +1,108 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Any, Tuple, Union
+
+
+class Feature:
+    def __init__(self,
+                 data: List[Any],
+                 pad_id: Union[int, List],
+                 dim: int):
+        """
+        Args:
+        data (List[Any]):
```

Comment:
Should we make documentation of these arguments to indent one more level?

Reply:
OK. As above, haven't carefully considered about documentation. Will take care of them from now.